coalib/testing/BaseTestHelper.py

```diff
@@ -0,0 +1,56 @@
+from coala_utils.Comparable import Comparable
+
+
+class BaseTestHelper:
+    """
+    This is a base class for all Bears' tests of coala's testing API.
+    """
+
+    def _assert_comparable_equal(self,
```

Comment:
module level scope means move it outside the class.

Reply:
Ohk. Got it.